### PR TITLE
Ability to open debug menu in editor

### PR DESCRIPTION
**Pull request by MatusGuy**

### CODE REVIEW

**Review by mrkubax10 — APPROVED** [review]

**Review by mrkubax10 — CHANGES_REQUESTED** [review]
I have just noticed that some builds are failing, please fix the issues.

**Comment by MatusGuy** [comment]
😁

File: src/supertux/menu/debug_menu.cpp [hunk 1]
```diff
@@ -76,6 +77,13 @@ DebugMenu::DebugMenu() :
   add_back(_("Back"));
 }
 
+DebugMenu::~DebugMenu() {
```
Comment (Vankata453) [inline src/supertux/menu/debug_menu.cpp:80]:
```
DebugMenu::~DebugMenu()
{
}
```

File: src/editor/editor.cpp [hunk 2]
```diff
@@ -683,7 +690,7 @@ Editor::setup()
     m_enabled = true;
     m_toolbox_widget->update_mouse_icon();
   }
-  
+
```
Comment (Vankata453) [inline src/editor/editor.cpp:693]: Empty lines at the ends of functions can be removed.

**Review by Vankata453 — APPROVED** [review]

File: src/editor/editor.cpp (hunk 2 above)
Comment (MatusGuy) [reply src/editor/editor.cpp:693]: my auto formatter did this

**Review by mrkubax10 — APPROVED** [review]
Apply changes which @Vankata453 suggested and then it will be fine.

**Pull request merged**